client/src/modules/chapters/graphql/queries.ts

```diff
@@ -80,6 +80,23 @@ export const CHAPTERS = gql`
       description
       category
       imageUrl
+      events {
+        id
+        name
+        description
+        start_at
+        invite_only
+        canceled
+        image_url
+        tags {
+          tag {
+            id
+            name
+          }
+        }
+        invite_only
+        canceled
```

Comment:
```suggestion
        start_at
```
No need to have other fields in query if they aren't used